src/experimental/DataBase.cpp

```diff
@@ -236,6 +251,20 @@ TableData DataBase::getEntryDataNew(DataPosition dataPosition)
         entryData.insert(column, model.record(0).value(column).toString());
     }
     return entryData;
+}*/
+
+Entry DataBase::getEntry(DataPosition dataPosition)
+{
+    Entry entry(dataPosition);
```

Comment:
Can i take this as a cue to stick to copy constructor initialisation? (insted of operator= copy)